recipes/libpdal/meta.yaml

```diff
@@ -22,13 +22,15 @@ requirements:
     - libtiff >=4.0.3,<4.1
     - proj4 4.9.3
     - libgdal
+    - curl 7.54.1
     - vc 9  # [win and py27]
     - vc 14  # [win and (py35 or py36)]
   run:
     - geos >=3.6
     - libtiff >=4.0.3,<4.1
     - proj4 4.9.3
     - libgdal
+    - curl 7.54.1
```

Comment:
we should probably put a range here

Reply:
In my experience, ranges don't work well, as you will upgrade your libraries just by recompile (if new libraries are available). So, you potentially from a package that compiles and works correctly to one that does not without changing anything. I would rather explicitly upgrade when needed. In this case, 7.55.1 did not work well for me (I think).